compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -410,6 +410,8 @@ prettyScenarioErrorError (Just err) =  do
           scenarioError_ContractIdInContractKeyKey
     ScenarioErrorErrorComparableValueError _ ->
       pure "Attend to compare incomparable values"
+    ScenarioErrorErrorTooDeepValueError _ ->
+          pure "Exceed maximum nesting value"
```

Comment:
```suggestion
          pure "Value exceeds maximum nesting value of 100"
```
you could include the 100 in the error to avoid duplicating it but I think we should show it in the error.